backend/docker.go

```diff
@@ -187,7 +189,12 @@ func newDockerProvider(cfg *config.ProviderConfig) (Provider, error) {
 		binds = strings.Split(cfg.Get("BINDS"), " ")
 	}
 
-	tmpFs := str2map(cfg.Get("TMPFS_MAP"))
+	securityOpt := []string{}
+	if cfg.IsSet("SECURITY_OPT") {
+		securityOpt = strings.Split(cfg.Get("SECURITY_OPT"), " ")
+	}
+
+	tmpFs := str2map(cfg.Get("TMPFS_MAP"), " ")
```

Comment:
@JaroslawSokol, how is it going to split a string from the example above? `/run:rw,exec+/run/lock:rw,exec`

Reply:
It is done by config/provider_config.go
System ENV variable like this KEY=VALUE_PART_1+VALUE_PART_2
is finally converted to provider config map seen as
"KEY" with value "VALUE_PART_1 VALUE_PART_2"